kafka/signed_token_redeem_handler.go

```diff
@@ -43,17 +45,19 @@ func SignedTokenRedeemHandler(
 	if len(tokenRedeemRequestSet.Data) > 1 {
 		// NOTE: When we start supporting multiple requests we will need to review
 		// errors and return values as well.
-		return fmt.Errorf("request %s: data array unexpectedly contained more than a single message. this array is intended to make future extension easier, but no more than a single value is currently expected", tokenRedeemRequestSet.Request_id)
+		message := fmt.Sprintf("request %s: data array unexpectedly contained more than a single message. This array is intended to make future extension easier, but no more than a single value is currently expected", tokenRedeemRequestSet.Request_id)
+		return utils.ProcessingErrorFromErrorWithMessage(err, message, logger)
 	}
 	issuers, err := server.FetchAllIssuers()
 	if err != nil {
-		return fmt.Errorf("request %s: failed to fetch all issuers: %w", tokenRedeemRequestSet.Request_id, err)
+		message := fmt.Sprintf("request %s: failed to fetch all issuers", tokenRedeemRequestSet.Request_id)
```

Comment:
Wouldn't this be a potential temporary error case? I think I only saw dynamo failures in `ErrorIsTemporary`, are there potential postgres errors that should also be covered?